### DISCUSSION TITLE
tfc-end2end example

### DISCUSSION THREAD

**Discussion by brsylmz23** [Xilinx/finn#392] [Q&A] [answered]
Hi all,

I got this error from the default codes on tfc_end2end example. Did someone know that why this is happening?

Thanks in advanc
![Ekran Alıntısı](https://user-images.githubusercontent.com/41904237/137018201-c7872b82-cc5d-4e6b-9139-3c9e2064d918.PNG)
e.

**Comment by heborras** [comment, marked as answer]
Hi @brsylmz23,
it looks like the `get_data` function has issues supplying the example data. Could you tell me what the variable `raw_i` contains? It looks like it might be `None`.
Just to check: Are you running FINN in the docker container? So with `bash ./run-docker.sh notebook` as described here: https://finn.readthedocs.io/en/latest/getting_started.html#running-finn-in-docker

**Reply by brsylmz23** [reply]
It is none. I checked it and I am running FINN in the docker container. I'm starting to think that the problem is with the location of PYNQ_TARGET_DIR. Because I don't have "finn_cnv_end2end_example" in this directory. Also my Xilinx folder is in tools folder(not in home folder).  Actually I don't know where I can find "finn_cnv_end2end_example".
![image](https://user-images.githubusercontent.com/41904237/137954948-fe3adeab-fda0-4b05-ba47-d8f44ef5a158.png)

**Reply by heborras** [reply]
That's most likely not the issue. The directory saved in the `PYNQ_TARGET_DIR` variable is the directory where the final results will be saved on the PYNQ board. So this has nothing to do with your local machine.

However, the issue which you are seeing is likely the same as discussed here: https://github.com/Xilinx/finn/issues/269
And the problem has since been fixed on both the master and dev branch.
Could you try running the `tfc_end2end example` notebook again, but this time on the most recent commit of the dev branch?

**Reply by brsylmz23** [reply]
Thank you, that solved my problem.
